test/cppsuite/configs/search_near_01_stress.txt

```diff
@@ -0,0 +1,22 @@
+# Configuration for search_near_01 stress test.
+# The test will generate key_count_per_collection number of keys for each prefix in aaa -> zzz.
+# This config will have a 30 minute duration, with 3 tables of an entry for each prefix.
+duration_seconds=1800,
+cache_size_mb=1000,
+timestamp_manager=
+(
+    enabled=false,
+),
+workload_generator=
+(
+    populate_config=
+    (
+        collection_count=3,
+        key_count_per_collection=100,
+        key_size=5,
+    ),
+    read_config=
+    (
+        thread_count=10
```

Comment:
@jiechenbo, should we increase the `op_rate` for the `read_operation` threads?

Reply:
I realise we call `tc->sleep();` to sleep on the read operation. Does op_rate configure the amount sleep? If so we definitely should!

Reply:
What should it be increased to?

Reply:
Like the other config, `10ms` should be good? At the end of the day, we need to check on T2 that the system is being stressed in the expected areas.

Reply:
@jiechenbo What do you mean? Larger value means a higher rate?

Reply:
Yes, this is what we mean, increase the rate of execution. The default value is `1s` as defined [here](https://github.com/wiredtiger/wiredtiger/blob/develop/dist/test_data.py#L47-L51), a smaller value for a higher rate. Sorry for the confusion.

Reply:
@etienneptl If smaller is a higher rate, then the rate is already increased.